recipes/coin-utils/all/conanfile.py

```diff
@@ -55,22 +58,15 @@ def layout(self):
     def requirements(self):
         self.requires("bzip2/1.0.8")
         self.requires("zlib/[>=1.2.11 <2]")
-        # TODO: add blas and lapack support
-
-    def validate(self):
-        if self.settings.os == "Windows" and self.options.shared:
-            raise ConanInvalidConfiguration("coin-utils does not provide a shared library on Windows")
-        # FIXME: This issue likely comes from very old autotools versions used to produce configure.
-        #        It might be fixed by calling autoreconf, but https://github.com/coin-or-tools/BuildTools
-        #        should be packaged and added to build requirements.
-        if hasattr(self, "settings_build") and cross_building(self) and self.options.shared:
-            raise ConanInvalidConfiguration("coin-utils shared not supported yet when cross-building")
+        self.requires("openblas/0.3.26")
```

Comment:
```suggestion
        self.requires("openblas/0.3.27")
```

Reply:
I'll handle this once the two prerequisite PRs have been merged.